var/spack/repos/builtin/packages/flecsi/package.py

```diff
@@ -29,6 +29,8 @@ class Flecsi(CMakePackage):
             description='Build FleCSI Tutorials')
 
     depends_on("cmake@3.1:",  type='build')
+    # Requires cinch > 1.0 due to cinchlog installation issue
+    depends_on("cinch@1.01:", type='build')
```

Comment:
Is this a typo? Should it be `1.0:`?

Reply:
To my recollection, ```1.0:``` includes version 1.0 which does not contain the required fix. Is there a cleaner way to provide an exclusive bound?

Reply:
Do you mean `1.0.1:` then?

Reply:
```1.01:``` better matches the current flecsi versioning scheme and ```1.0.1:``` seems to hit a bug (feature?) in spack's concretizer that results in

```==> Error: Invalid Version range: 1.0.1:1.0``` which seems related to issue  #9744

Reply:
You mean the current `cinch` versioning scheme? I'm not sure what you're trying to do here. If you want `cinch > 1.0`, that would look like `cinch@1.0.1:`. If you want `cinch >= 2.0`, that would look like `cinch@2:`.

Reply:
Sorry. Cinch is part of the same project flecsi is under. I am referring to the current overall scheme of the project (to my understanding)

And, regardless, as mentioned in the issue I linked you to, as well as from testing, ```cinch@1.0.1:``` won't work due to a concretization issue. Likely because ```cinch@1.0``` is not ```cinch@1.0.1```

Is this a blocker to merge this PR in?

Reply:
It looks like `cinch@1.0` is the newest release? So if you need `cinch > 1.0`, does that mean the master branch is required?

Reply:
This "Requires cinch > 1.0 due to cinchlog installation issue"

That has been fixed on the master/development branch. It will be included in the next cinch release. Rather than update the flecsi spackage (and all other spackages that depend on cinch) to switch from the develop version back to a tagged release, we will require a version > 1.0.

Currently, to my knowledge, the project that cinch and flecsi is part of is more likely to follow 1.0 with 1.01 or even 1.1 rather than 1.0.1. That is subject to change, but it is also a moot point because Issue #9744 is a problem here where spack can't compare 1.0 and 1.0.1


Please let me know what solution you would like me to use